controllers/discovery/initDiscoveryUserData.js

```diff
@@ -129,7 +136,8 @@ const InitDiscoveryUserData = async (req, res) => {
         limit: limit,
         offset: page * limit,
         minimumKarmaScore: MINIMUM_KARMA_SCORE,
-        admin_user_id: process.env.BETTER_ADMIN_ID
+        admin_user_id: process.env.BETTER_ADMIN_ID,
+        excluded_users: excluded_users?.splice(0, MAX_EXCLUDED_USERS)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `admin_user_id` and `excluded_users` to the query parameters is done correctly. However, the use of `splice` directly on `excluded_users` will mutate the original array. It's better to use `slice` instead, which returns a new array and does not modify the original one.
```diff
- excluded_users: excluded_users?.splice(0, MAX_EXCLUDED_USERS)
+ excluded_users: excluded_users?.slice(0, MAX_EXCLUDED_USERS)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->